DxOper.pas

```diff
@@ -141,7 +144,7 @@ function TDxOperator.IsMyCall: TCallCheckResult;
 
       D := M[x-1,y-1];
       //'?' matches any char
-      if not (C[x] in [C0[y], '?']) then Inc(D, W_D);
+      if not CharInSet(C[x], [C0[y], '?']) then Inc(D, W_D); //grg1

```

Comment:
Callsign comparison does non need to be Unicode, declare C and C0 as AnsiString than you willo not have to change this line